irods/test/login_auth_test.py

```diff
@@ -118,7 +118,8 @@ class TestLogins(unittest.TestCase):
          Per https://slides.com/irods/ugm2018-ssl-and-pam-configuration#/3/7
 
       4. Create rodsuser alissa and corresponding unix user with the appropriate
-         passwords as below.
+         passwords as below.  For test_0[1-4] and test[1-4] to pass, user alissa must
+         have the native iRODS password set, ie.: 'iadmin moduser alissa password apass'.
```

Comment:
For 4.3.0+, the pam authentication scheme should be updated to pam_password (see line 131 and 162).

I'm also a little surprised that it's required to have a native authentication password set, but I don't see a need to look into this right now.

Reply:
Please disregard.